.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -18,5 +18,7 @@ A clear and concise description of any alternative solutions or features you've
 
 Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc
 
+- [ ] This feature is expected to need manual testing.
```

Comment:
Should we add checkbox for things that need a dev a11y swarm here too, following @greenc05's suggestion?